music/live-coding-ksql-music.adoc

```diff
@@ -110,9 +110,13 @@ To register the topic `play-events`, from the KSQL query editor:
 CREATE STREAM ksql_playevents WITH (KAFKA_TOPIC='play-events', VALUE_FORMAT='AVRO');
 ----
 
-Or from Control Center in the KSQL Streams view, select "Create Stream" and fill out the fields as shown below.  Because of Control Center integration with Confluent Schema Registry, it automatically detects the fields `song_id` and `duration` and their respective data types.
+Or from Control Center in the KSQL Streams view, select "Add a stream".
 
-image:images/ksql_playevents.png[play-events]
+image::images/add_a_stream.png[add-stream] 
+
+Fill out the fields as shown below.  Because of Control Center integration with Confluent Schema Registry, it automatically detects the fields `song_id` and `duration` and their respective data types.
```

Comment:
```suggestion
Fill out the fields as shown below.  Because of Control Center integration with Confluent Schema Registry, KSQL automatically detects the fields `song_id` and `duration` and their respective data types.
```